src/partials/60_section-gallery.html

```diff
@@ -0,0 +1,329 @@
+                  data-toggle="tooltip"
+                  data-placement="right"
+                  title="Add To Compare"
+                  ><i class="fas fa-exchange-alt"></i
+                ></a>
+                <a
+                  href="#"
+                  class="btn-outline"
+                  data-toggle="tooltip"
+                  data-placement="right"
+                  title="Quick View"
+                  ><i class="far fa-eye"></i
+                ></a>
+                <a
+                  href="#"
+                  class="btn-outline"
+                  data-toggle="tooltip"
+                  data-placement="right"
+                  title="Add To Cart"
+                  ><i class="fa fa-shopping-basket"></i
+                ></a>
+              </div>
+              <div class="content-wrapper">
+                <div class="content">
+                  <h5>Aenean Ru Bristique</h5>
+                  <div class="stars">
+                    <a href="#" class="full"></a> <a href="#" class="full"></a>
+                    <a href="#"></a> <a href="#"></a> <a href="#"></a>
+                  </div>
+                </div>
+                <div class="price">
+                  <p>$120.00</p>
+                  <p class="old-price"><s>$160.00</s></p>
+                </div>
+              </div>
+            </div>
+            <div
+              class="tab-pane fade top-seller show active"
+              id="nav-top-seller"
+              role="tabpanel"
+              aria-labelledby="nav-top-seller-tab"
+              style="background-image: url(images/gallery/chair-2.png);"
+            >
+              <div class="outlines">
+                <a
+                  href="#"
+                  class="btn-outline"
+                  data-toggle="tooltip"
+                  data-placement="right"
+                  title="Add To Wishlist"
+                  ><i class="far fa-heart"></i
+                ></a>
+                <a
+                  href="#"
+                  class="btn-outline"
+                  data-toggle="tooltip"
+                  data-placement="right"
+                  title="Add To Compare"
+                  ><i class="fas fa-exchange-alt"></i
+                ></a>
+                <a
+                  href="#"
+                  class="btn-outline"
+                  data-toggle="tooltip"
+                  data-placement="right"
+                  title="Quick View"
+                  ><i class="far fa-eye"></i
+                ></a>
+                <a
+                  href="#"
+                  class="btn-outline"
+                  data-toggle="tooltip"
+                  data-placement="right"
+                  title="Add To Cart"
+                  ><i class="fa fa-shopping-basket"></i
+                ></a>
+              </div>
+              <div class="content-wrapper">
+                <div class="content">
+                  <h5>Aenean Ru Bristique</h5>
```

Comment:
Przydałoby się nadać temu jakąś klasę - idziemy ku temu, żeby elementy html nie zostawiać bez klas

Reply:
Nadałam klasę `product-title`. Ponieważ ten fragment kodu był skopiowany z `50_section-products.html`, to musiałam nanieść zmiany również tam.